tests/cases/fourslash/completionForObjectProperty.ts

```diff
@@ -0,0 +1,38 @@
+/// <reference path="fourslash.ts" />
+
+// @Filename: /a.ts
+//// export const foo = { bar: 'baz' };
+
+// @Filename: /b.ts
+//// const test = foo/*1*/
+
+// @Filename: /c.ts
+//// const test2 = {...foo/*2*/}
+
+// @Filename: /d.ts
+//// const test3 = [{...foo/*3*/}]
+
+// @Filename: /e.ts
+//// const test4 = { foo/*4*/ }
```

Comment:
Are there tests for `{ foo: /**/ }` and `{ unrelated: foo/**/ }`? I'd expect the first *not* to import and the second to import.
If not, please add them.

Reply:
I don’t think I understand why you wouldn’t expect to see an import on the former 🤔

Reply:
I added some test cases. Currently, both cases will import. Do you think it meets expectations?

Reply:
Err, I really underspecified that. Sorry. What I meant was that I didn't expect `{ foo: /*5*/ }` to import `foo`, since it's a property name. It makes sense to import whatever is provided for the property value.

I guess the test case I should have asked for is `{ foo/*7*/: "unrelated" }`, but now I suspect that existing test cases cover this already.

Reply:
Yeah, `{ foo/*7*/: "unrelated" }` is currently covered.